distribution/src/assemble/component.xml

```diff
@@ -245,10 +245,6 @@
       <source>../NOTICE</source>
       <outputDirectory />
     </file>
-    <file>
-      <source>../git.properties</source>
```

Comment:
Let's leave this file to be placed in the target distribution.

Reply:
@vvysotskyi thanks that slipped through because I decided to start from the existing commit in the original PR for this.